wishlistApp/views.py

```diff
@@ -75,8 +75,28 @@ def delete_account(request):
 
 @login_required
 def user_list(request):
-    return render(request, 'wishlistApp/userList.html')
+    user = request.user.get_username()
+    return render(request, 'wishlistApp/userList.html', {'user':user})
 
 @login_required
 def new_item(request):
-    return render(request, 'wishlistApp/newItem.html')
+    if request.method == 'POST':
+        form = ItemUpdateForm(request.POST, instance=request.user)
+        if form.is_valid():
+            form.save()
+            #grabbing info user typed in form to save into an Item obj
+            #(not sure if saving the form OR putting info into an Item obj is correct)
+            '''
+            name = request.POST['itemN']
+            url = request.POST['itemURL']
+            desc = request.POST['itemD']
+            image = request.POST['imageURL']
+            priority = request.POST['itemP']
+            ins = Item(name=name, image=url, description=desc, priority = priority, user_id=request.user.get_username())
+            ins.save()
+            '''
+            messages.success(request, f"Item has been added to your Wishlist!")
+            context = {
+                'form' : form,
+            }
+            return render(request, 'wishlistApp/newItem.html', context)
```

Comment:
Your current add-item route does not work because you only have POST functionality. Not sure if you're still working on it, but make sure you add GET functionality too so that a form is displayed. POST functionality only deals with the form after it's been submitted, so it won't display the form if you don't include GET stuff too.